src/java.base/share/classes/javax/net/ssl/KeyStoreBuilderParameters.java

```diff
@@ -68,8 +68,7 @@ public KeyStoreBuilderParameters(List<Builder> parameters) {
             throw new IllegalArgumentException();
         }
 
-        this.parameters = Collections.unmodifiableList(
-            new ArrayList<Builder>(parameters));
+        this.parameters = List.copyOf(parameters);
```

Comment:
Hello Mark, this would actually be a change in behaviour. The `List.copyOf` says:

> The given Collection must not be null and it must not contain any null elements.

The current implementation of the public constructor on the public `KeyStoreBuilderParameters` mandates no such requirement. So if there's some code which currently passes a list with a null element in it, then this change will now end up throwing a `NullPointerException` as per the contract of `List.copyOf`.

Reply:
You are correct. This is not a good change since it changes behavior. Going back to the original. Thanks for your review!